### PR TITLE
Special Buildings

### PR DESCRIPTION
**Pull request by Gaben1025**
## Changelog:
- Added new Special Buildings
    - Gurubashi Arena
    - Karazhan
    - Sunwell
    - Lordaeron/Ruined Palace
    - Temple of Atal'Hakar
    - Thoradins Wall (2 Tiers)
    -  Violet Citadel
- Added the buildings to their respective provinces

- [x] Add loc for all buildings
- [x] Add all buildings on the spreadsheet for Eastern Kingdoms
- [x] Custom assets later?

Ideas pulled from: https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1906407470

### CODE REVIEW

**Review by TheGroxEmpire — COMMENTED** [review]
- The stats given to these special buildings aren't matching the values on the sheet:
- - Karazhan:
<details>

image
image

</details>

- - Ruined Palace / Lordaeron Palace:
<details>

image

image
image
</details>

- - Sunwell:
<details>

image
image

</details>

- - Violet Citadel (Missing Garisson):
<details>

image
image

</details>

- I can't find Thoradin's wall in Arathi Highlands - Hillsbrad border

**Review by TheGroxEmpire — APPROVED** [review]
My concerns were fixed.

File: localization/english/wc_buildings_l_english.yml [hunk 1]
```diff
@@ -1,11 +1,55 @@
...
  building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds of Azeroth and Draenor, allowing alien creatures to go back and forth."
  building_dark_portal_01:1 "Dark Rift"
  building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds of Azeroth and Draenor."
  building_dark_portal_02:1 "Dark Portal"
  building_dark_portal_02_desc:1 "A massive arch with magical runes that maintains the connection between Azeroth and Draenor, allowing large armies to pass through the portal."
 
+ building_type_gurubashi_arena_01:0 "Gurubashi Arena"
+ building_type_gurubashi_arena_01_desc:1 "An arena for prisoners of war, kidnapped travelers, those in debt, and even some brave fools who fight for their freedom or for the spoils of victory."
+ building_gurubashi_arena_01:1 "Gurubashi Arena"
+ building_gurubashi_arena_01_desc:1 "The Gurubashi Arena provides ruffians a shot at competition, fame, and treasure. It is said that in ancient times, fights were to the death, and the sand of the area was muddy with troll blood."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:13]:
```suggestion
 building_gurubashi_arena_01_desc:1 "The Gurubashi Arena provides ruffians a shot at competition, fame, and treasure. It is said that in ancient times, fights were to the death, and the sand of the arena was muddy with troll blood."
```

File: localization/english/wc_buildings_l_english.yml [hunk 2]
```diff
@@ -1,11 +1,55 @@
...
+ 
+ building_type_karazhan_01:0 "Karazhan"
+ building_type_karazhan_01_desc:1 "A massive tower that watches over Deadwind Pass. Its inhabitants wield magical powers coveted by all of Azeroth."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:16]:
```suggestion
 building_type_karazhan_01_desc:1 "A massive tower that watches over Deadwind Pass. Its inhabitants wield magical powers coveted by every denizen of Azeroth."
```

I think it sounds fancier, but this is me being very pedantic.

File: localization/english/wc_buildings_l_english.yml [hunk 3]
```diff
@@ -1,11 +1,55 @@
...
+ building_karazhan_01:1 "Karazhan"
+ building_karazhan_01_desc:1 "Also known as "Medivh's Tower" or "Ivory Tower of Karazhan", Karazhan is an abandoned citadel located on a nexus of ley lines in southern Deadwind Pass. The tower is best known for its last known occupant Medivh, the last Guardian of Tirisfal."
+ 
+ building_type_lordaeron_palace_01:0 "Lordaeron Palace"
+ building_type_lordaeron_palace_01_desc:1 "A palace that holds together the domain of its ruler. It is a beacon of prosperity and unity to all who lay eyes upon it." 
+ building_lordaeron_palace_01:1 "Lordaeron Palace"
+ building_lordaeron_palace_01_desc:1 "The Palace of Lordaeron is the jewel of the Empire of Lordaeron and shows just how far humans have come. Its presence exudes an aura of prestige and with a hint of decadence."
+ 
+ building_type_temple_of_atal_hakkar_01:0 "Temple of Atal'Hakar"
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:25]:
```suggestion
 building_type_temple_of_atal_hakkar_01:0 "Temple of Atal'Hakkar"
```

I believe it's spelled [with two k's](https://wowpedia.fandom.com/wiki/Temple_of_Atal%27Hakkar)?

File: localization/english/wc_buildings_l_english.yml [hunk 4]
```diff
@@ -1,11 +1,55 @@
...
+ building_type_temple_of_atal_hakkar_01_desc:1 "A Temple built to bring back an ancient blood god into Azeroth. A massive shrine to his greatness."
+ building_temple_of_atal_hakkar_01:1 "Temple of Atal'Hakar"
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:27]:
```suggestion
 building_temple_of_atal_hakkar_01:1 "Temple of Atal'Hakkar"
```

File: localization/english/wc_buildings_l_english.yml [hunk 5]
```diff
@@ -1,11 +1,55 @@
...
+ building_temple_of_atal_hakkar_01_desc:1 " An influential group of troll priests, known as the Atal'ai, seek to bring back an ancient blood god named Hakkar the Soulflayer. They erected this great temple to Hakkar - where they could prepare for his arrival into the physical world."
+ 
+ building_type_thoradins_wall_01:0 "Thoradin's Wall"
+ building_type_thoradins_wall_01_desc:1 "Colossal walls that sheild the denzins of its holders domain. All unwanted guests must firstly face this almost insurmountable obstacle."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:31]:
```suggestion
 building_type_thoradins_wall_01_desc:1 "Colossal walls that shield the denizens of its holder's domain. All unwanted guests must first face this almost insurmountable obstacle."
```

Spelling errors.

File: localization/english/wc_buildings_l_english.yml [hunk 6]
```diff
@@ -1,11 +1,55 @@
...
+ building_thoradins_wall_01:1 "Ruined Thoradin's Wall"
+ building_thoradins_wall_01_desc:1 "Once a wonder for our people, a testament to our power and our technological prowess, Thoradin's Wall now only statnds as a symbol of our downfall. Some parts of the wall still remain and help keep the peace, but the mostly ruined wall is but a shell of the once great shield against those who would wish harm upon us."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:33]:
```suggestion
 building_thoradins_wall_01_desc:1 "Once a wonder for our people, a testament to our power and our technological prowess, Thoradin's Wall now only stands as a symbol of our downfall. Some parts of the wall still remain and help keep the peace, but the mostly ruined wall is but a shell of the once great shield against those who would wish harm upon us."
```
Spelling error.

File: localization/english/wc_buildings_l_english.yml [hunk 7]
```diff
@@ -1,11 +1,55 @@
...
+ building_thoradins_wall_02:1 "Thoradin's Wall"
+ building_thoradins_wall_02_desc:1 "In its full glory, Thoradin's Wall stands proudly as a bastion for our people. We are protected by the towering walls and the brave soldiers guarding us from the dangers outside."
+ 
+ building_type_ruined_palace_01:0 "Ruined Palace"
+ building_type_ruined_palace_01_desc:1 "A city that spans beneath much of Lordaeron's ruin. A grave to kingdom."
+ building_ruined_palace_01:1 "Ruined Palace"
+ building_ruined_palace_01_desc:1 "The Ruins of Lordaeron is all that remains of the city of Lordaeron, the great capital city of the Kingdom of Lordaeron. Now it serves as a sad reminder of the power of the Scourge. The Ruined Palace still stands and its inhabitants are constantly haunted by the final tragic events that led to the death of King Terenas and the city's destruction."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:40]:
```suggestion
 building_ruined_palace_01_desc:1 "The Ruins of Lordaeron is all that remains of the city of Lordaeron, the great capital city of the Kingdom of Lordaeron. Now it serves as a grim reminder of the power of the Scourge. The Ruined Palace still stands and its inhabitants are constantly haunted by the final tragic events that led to the death of King Terenas and the city's destruction."
```

I think grim sounds a bit better, but this is just a suggestion.

File: localization/english/wc_buildings_l_english.yml [hunk 8]
```diff
@@ -1,11 +1,55 @@
...
+ 
+ building_type_sunwell_01:0 "The Sunwell"
+ building_type_sunwell_01_desc:1 "A fountain of mystical power located in the Sunwell Plateau on the Isle of Quel'Danas. Its presence in its users lives is comparable to eating or breathing."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:43]:
```suggestion
 building_type_sunwell_01_desc:1 "A fountain of mystical power located within the Sunwell Plateau on the Isle of Quel'Danas. Its users rely on its energies for sustenance."
```

I think this part of the sentence sounded a bit confusing. Not sure if I made it better or worse though.

File: localization/english/wc_buildings_l_english.yml [hunk 9]
```diff
@@ -1,11 +1,55 @@
...
+ building_sunwell_01:1 "The Sunwell"
+ building_sunwell_01_desc:1 "The Sunwell has empowered us high elves for roughly seven thousand years. Its presence in their our is comparable to eating or breathing. In the Sunwell's light, Quel'Thalas has grown mighty, becoming a beacon of power in the Eastern Kingdoms."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:45]:
```suggestion
 building_sunwell_01_desc:1 "The Sunwell has empowered us high elves for roughly seven thousand years. Its presence in our lives is akin to eating or breathing. In the Sunwell's light, Quel'Thalas has grown mighty, becoming a beacon of power in the Eastern Kingdoms."
```
A grammar error and a small pedantic suggestion.

File: localization/english/wc_buildings_l_english.yml [hunk 10]
```diff
@@ -1,11 +1,55 @@
...
+
+ building_type_violet_citadel_01:0 "Violet Citadel"
+ building_type_violet_citadel_01_desc:1 "A Citadel working as both a place of study and progress as well as a bastion houseing great casters to defend against invading forces."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:48]:
```suggestion
 building_type_violet_citadel_01_desc:1 "A Citadel working as both a place of study and progress as well as a bastion housing great casters to defend against invading forces."
```
Spelling error.

File: localization/english/wc_buildings_l_english.yml [hunk 11]
```diff
@@ -1,11 +1,55 @@
...
+ building_violet_citadel_01:1 "Violet Citadel"
+ building_violet_citadel_01_desc:1 "The Violet Citadel is the main fortress of Dalaran which houses the Chamber of Air, research laboratories and great spell libraries of the Kirin Tor."
```
Comment (ercarp) [inline localization/english/wc_buildings_l_english.yml:50]:
```suggestion
 building_violet_citadel_01_desc:1 "The Violet Citadel is the main fortress of Dalaran which houses the Chamber of Air, research laboratories, and the great spell libraries of the Kirin Tor."
```

I would add an Oxford comma here for readability. 😄

**Review by ercarp — CHANGES_REQUESTED** [review]
There are a few spelling errors that need to be fixed, but other than that it looks good. I included a few minor suggestions as well but you can take them or leave them.

**Review by Gaben1025 — COMMENTED** [review]
.

**Review by ercarp — APPROVED** [review]
On behalf of @Warcraft-GoA-Development-Team/english-proofreaders, localization looks good to me! Didn't review other areas of the PR.

**Review by arithon — CHANGES_REQUESTED** [review]
just a couple remarks that might be taken into consideration.

**Review by arithon — APPROVED** [review]

File: common/buildings/wc_special_buildings.txt [hunk 12]
```diff
@@ -42,5 +42,194 @@ dark_portal_02 = {
...
+	province_modifier = {
+		hostile_raid_time = 4
+		fort_level = 6
+	}
+
+	character_modifier = {
+		learning = 3
+		monthly_lifestyle_xp_gain_mult = 0.4
+		monthly_piety_gain_mult = 0.2
+		caster_toughness_mult = 0.7
```
Comment (zumbak04) [inline common/buildings/wc_special_buildings.txt:79]: I would nerf it... looks too strong. Holy places give much less.

**Comment by Gaben1025** [comment]
<https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1906407470>

File: common/buildings/wc_special_buildings.txt (hunk 12 above)
Comment (zumbak04) [reply common/buildings/wc_special_buildings.txt]: I would say other buildings need nerf too. If you use the balance spreedsheet, I should aim at lesser cost number.

Comment (zumbak04) [reply common/buildings/wc_special_buildings.txt]: 40-30 prob.

**Review by zumbak04 — APPROVED** [review]
I'll hit approve, but left comments.

**Pull request merged**